### PR TITLE
ref(debug-files): Split into smaller pages

### PR DESCRIPTION
**Pull request by jan-auer**
Splits the "Debug Information Files" platform page into smaller sub-pages. Reprocessing has moved to _Product > Error Monitoring_. There is no change to the content of the pages. This is the first step of https://github.com/getsentry/sentry-docs/issues/2872

@PeloWriter Please review the descriptions of the new sub pages. You do not have to read the content (yet), as it has not changed from its previous version.

image

### CODE REVIEW

**Review by marandaneto — APPROVED** [review]
LGTM

File: src/docs/product/error-monitoring/reprocessing.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Reprocessing
+sidebar_order: 10
+description: Sentry can reprocess issues that were missing required debug information files. Learn which platforms support reprocessing and how to set it up for your projects.
```
Comment (PeloWriter) [inline src/docs/product/error-monitoring/reprocessing.mdx:4]:
```suggestion
description: Learn about reprocessing, which can occur for issues missing required debug information files, including which platforms support reprocessing and how to set it up for your projects.
```

File: src/platforms/common/data-management/debug-files/file-formats.mdx [hunk 2]
```diff
@@ -0,0 +1,203 @@
+---
+title: "Debug File Formats"
+sidebar_order: 1
+description: "Learn about the platform-specific file formats and the debug information they contain."
```
Comment (PeloWriter) [inline src/platforms/common/data-management/debug-files/file-formats.mdx:4]:
```suggestion
description: "Learn about platform-specific file formats and the debug information they contain."
```

File: src/platforms/common/data-management/debug-files/file-formats.mdx [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+---
+
+Sentry differentiates four kinds of debug information:
+
+- **Debug Information:** Provides function names, paths to source files, line
+  numbers and inline frames. The process of resolving this information from
```
Comment (PeloWriter) [inline src/platforms/common/data-management/debug-files/file-formats.mdx:10]:
```suggestion
  numbers, and inline frames. The process of resolving this information from
```

File: src/platforms/common/data-management/debug-files/file-formats.mdx [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+  exactly the same as executables.
+
+- **Debug Companions** are stored in `.pdb` files. They usually contain debug
+  information and in most cases symbol tables. For 32-bit programs, they also
+  contain unwind information. In rare cases, they might have different file
+  names than their corresponding executable.
+
+<Note>
+
+At the moment, Sentry supports Native PDBs only. PDBs for the .NET platform are not supported.
```
Comment (PeloWriter) [inline src/platforms/common/data-management/debug-files/file-formats.mdx:153]:
```suggestion
Currently, Sentry supports only Native PDBs. PDBs for the .NET platform are not supported.
```

File: src/platforms/common/data-management/debug-files/file-formats.mdx [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+
+</Note>
+
+## Breakpad Symbols
+
+The Google Breakpad library has established a platform-independent ASCII format
+to store debug information. Such files are usually generated for applications
+using Breakpad, Crashpad or the Electron Framework.
```
Comment (PeloWriter) [inline src/platforms/common/data-management/debug-files/file-formats.mdx:161]:
```suggestion
using Breakpad, Crashpad, or the Electron Framework.
```

File: src/platforms/common/data-management/debug-files/identifiers.mdx [hunk 6]
```diff
@@ -0,0 +1,120 @@
+---
+title: Debug Identifiers
+sidebar_order: 2
+description: "Learn about build tooling requirements to allow Sentry to uniquely identify debug information files across builds."
```
Comment (PeloWriter) [inline src/platforms/common/data-management/debug-files/identifiers.mdx:4]:
```suggestion
description: "Learn about build tooling requirements that allow Sentry to uniquely identify debug information files across builds."
```

File: src/platforms/common/data-management/debug-files/source-context.mdx [hunk 7]
```diff
@@ -0,0 +1,19 @@
+---
+title: Source Context
+sidebar_order: 5
+description: "Learn about setting up source bundles to show source code in stack traces on the issue details page."
```
Comment (PeloWriter) [inline src/platforms/common/data-management/debug-files/source-context.mdx:4]:
```suggestion
description: "Learn about setting up source bundles to show source code in stack traces on the Issue Details page."
```

File: src/platforms/common/data-management/debug-files/symbol-servers.mdx [hunk 8]
```diff
@@ -0,0 +1,251 @@
...
+sidebar_order: 4
+description: "Learn about symbol servers and how to configure them with your Sentry projects."
+---
+
+Sentry can download debug information files from external repositories. This
+allows you to stop uploading debug files and instead configure a public symbol
+server or run your own. It is also possible to configure external repositories
+and upload debug files at the same time.
+
+To configure external repositories, go to _Project Settings > Debug Files_.
```
Comment (PeloWriter) [inline src/platforms/common/data-management/debug-files/symbol-servers.mdx:12]: Bold path names (not italics):
```suggestion
To configure external repositories, go to **Project Settings > Debug Files**.
```

File: src/platforms/common/data-management/debug-files/upload/index.mdx [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+Files can be uploaded using the `upload-dif` command. This command will scan a
+given folder recursively for files and upload them to Sentry:
+
+<PlatformContent includePath="upload-dif" />
+
+For all available options and more information refer to [_Uploading Debug
+Information_](/product/cli/dif/#uploading-files).
+
+Always ensure that debug files are uploaded before deploying or releasing your
```
Comment (PeloWriter) [inline src/platforms/common/data-management/debug-files/upload/index.mdx:22]:
```suggestion
Debug files should be uploaded before deploying or releasing your
```

File: src/platforms/common/data-management/debug-files/upload/index.mdx [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+application so that crash reports can be processed. For manual testing, use the
+`sentry-cli upload-dif --wait` before sending the first native crash or error
+event.
+
+If you upload a file that was previously reported as missing by Sentry in a crash
+report, it may take up to one hour until Sentry starts using this file for new
+crash reports. Existing events and issues are not processed again.
+
+In Sentry, your uploaded files are associated with projects. You can view and
+manage uploads at _Project Settings > Debug Files_. This screen displays the
```
Comment (PeloWriter) [inline src/platforms/common/data-management/debug-files/upload/index.mdx:32]:
```suggestion
manage uploads at **Project Settings > Debug Files**. This screen displays the
```

**Review by PeloWriter — APPROVED** [review]
Minor edits for smoothing.

**Pull request merged**